src/lib/ipmi.c

```diff
@@ -79,7 +79,7 @@ int ipmicmd(struct led_ctx *ctx, int sa, int lun, int netfn, int cmd, int datale
 	memset(&raddr, 0, sizeof(raddr));
 	memset(&req, 0, sizeof(req));
 	memset(&rcv, 0, sizeof(rcv));
-	memset(tresp, 0, sizeof(uint8_t) + (resplen + 1));
+	memset(tresp, 0, sizeof(uint8_t) * (resplen + 1));
```

Comment:
Can you remove those memsets using (instruction supported from gcc 4 if I remember correctly):

```c
struct ipmi_system_interface_addr saddr = {0};
```
Reference I found:
https://stackoverflow.com/questions/2884230/zeroing-out-memory

If that is not possible, please use ARRAY_SIZE macro for getting a size of array.